sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -393,6 +398,18 @@ def test_global_fanout(self):
           | beam.CombineGlobally(combine.MeanCombineFn()).with_fanout(11))
       assert_that(result, equal_to([49.5]))
 
+  @attr('ValidatesRunner')
+  def test_hot_key_combining_with_accumulation_mode(self):
+    with TestPipeline() as p:
+      result = (p
+                | beam.Create([1, 2, 3, 4, 5])
+                | beam.WindowInto(GlobalWindows(),
+                                  trigger=Repeatedly(AfterCount(1)),
```

Comment:
We would need to use a test stream to make this test non-trivial.

Reply:
I think the same as you. We need at least Timestamped values.